### ISSUE TITLE
Edge vectors

### ISSUE THREAD

**Issue by meteoswiss-mdr** [pySTEPS/pysteps#2]
Adding pre-defined velocity vectors before the Interpolation:
Uli suggested adding COSMO-derived wind velocity vectors in the four edges of the domain (where there is no radar coverage), in order to force the interpolation towards the wind field of the model. I suggest providing an additional optional kwarg

```
edge_vec : array-like
        Wind vectors in 2d array (rows: x,y,u,v) obtained from model output (e.g. COSMO) to
        stabilise interpolation at image edges with no radar coverage.

```
which is then appended to the declustered motion vector array:

```
    # decluster sparse motion vectors
    x, y, u, v = declustering(x0, y0, u, v, decl_grid, min_nr_samples)


    # Append edge vectors if provided
    if not(edge_vec is None):
        x = [x,edge_vec[0,:]]; y = [y,edge_vec[1,:]]
        u = [u,edge_vec[2,:]]; v = [v,edge_vec[3,:]]

    # kernel interpolation
    X, Y, UV = interpolate_sparse_vectors(x, y, u, v, domain_size,
                                          epsilon=kernel_bandwidth, nchunks=nchunks)

```

https://github.com/pySTEPS/pysteps/blob/c12287fd32b4e1d0af9daa0e83f34f325efa82bc/pysteps/optflow/lucaskanade.py#L144

**Comment by dnerini** [comment]
@meteoswiss-mdr your suggestion was included in commit 2092249d24be39d60ac96f84fceef1ef409f04dd. 
Please have a look, I will close this issue once you confirm that the code is working as you intended.

**Comment by meteoswiss-mdr** [comment]
Great, thanks!
